src/azure-openapi-linter/tests/resources/ArraySchemaWithoutItems.json

```diff
@@ -0,0 +1,77 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Operations schemes has non-http/https type",
+        "description": "Some documentation.",
+        "version": "2014-04-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "basePath": "/",
+    "paths": {
+        "/foo": {
+            "post": {
+                "operationId": "Models_list",
+                "summary": "Foo path",
+                "description": "Foo operation",
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Models"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "test subscription id"
+        },
+        "ApiVersion": {
+            "name": "api-version",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "test api version"
+        }
+    },
+    "definitions": {
+        "Models": {
+            "description": "the models to return",
+            "properties": {
+                "mymodels": {
+                    "type": "array",
+                    "items": {
```

Comment:
what's the violation of the rule for this model? isn't "items" defined for the array type?